core/arch/arm/kernel/thread_spmc.c

```diff
@@ -24,10 +24,14 @@
 #include <string.h>
 #include <sys/queue.h>
 #include <tee/entry_std.h>
+#include <tee/uuid.h>
 #include <util.h>
 
 #include "thread_private.h"
 
+#define FFA_PARTITION_DIRECT_REQ_RECV_SUPPORT BIT(0)
```

Comment:
These two should preferably go into `core/arch/arm/include/ffa.h`

Reply:
Done